docs/componentExtension.md

```diff
@@ -0,0 +1,83 @@
+## Customizing and Extending Components
+
+As mentioned in the [architecture document](architecture.md), when customizing or extending the sample app component, do not modify the packages within `@sfcc-bff` and `@sfcc-core`. These packages will be published and consumed via `npm`. Instead, create a new custom package within the monorepo that registers itself with `@sfcc-core\core` and provides access to data from a third-party service.
+
+This example below illustrates how to create a product recommendations extension for product details component:
+
+Background : Product Details Page displays product name, product id, color swatches, images, price, etc. Our goal is to extend this component to display this product's recommendations on Product Detail Page. 
```

Comment:
recommendations on Product Detail Page.  => recommendations on the Product Detail Page.